cli/cmd/admin/start.go

```diff
@@ -168,9 +169,10 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 			group, cctx := errgroup.WithContext(ctx)
 
 			// Determine services to run. If no service name was provided, run them all.
-			// We just have two currently, so keeping this basic.
+			// We just have three currently, so keeping this basic.
 			runServer := len(args) == 0 || args[0] == "server"
 			runWorker := len(args) == 0 || args[0] == "worker"
+			runJobs := len(args) == 0 || args[0] == "jobs"
```

Comment:
Yeah, mainly ad-hoc use cases. But it won't be triggered from the command-line since it's not *submitting* a job, it's actually running it. It will run as a Kubernetes job. So there's no difference between configuring the job name and all the other config.

In the future, it would be much easier to be able to submit a job from the local CLI and have it routed to a worker, but for that we would need a task queue the workers can listen on, which we haven't set up yet